README.md

```diff
@@ -51,13 +58,13 @@ npm i
 - Bundle the typescript in src/main and src/renderer in `/build` and use those to build the electron app installer. The final executable will be found in `/dist`

```

Comment:
Mentioning Electron, has the doc been updated?

Reply:
Also, it is really slow on WSL.

Reply:
I've added a note to tell Windows users not to use WSL